packages/loader/src/meta.ts

```diff
@@ -42,13 +42,19 @@ export const npmPackages = {
     "dagre": "dagre/dist/dagre",
     "@deck.gl/core": "@deck.gl/core/dist",
     "@deck.gl/layers": "@deck.gl/layers/dist",
+    "@observablehq/parser": "@observablehq/parser/dist/parser.min",
+    "@observablehq/runtime": "@observablehq/runtime/dist/runtime.umd",
+    "@observablehq/inspector/dist/inspector.css": "@observablehq/inspector/dist/inspector.css",
+    "acorn": "acorn/dist/acorn",
+    "acorn-walk": "acorn-walk/dist/walk",
```

Comment:
What will we use `acorn` for?

Reply:
We don't but its pulled in by observable...